src/web/app/pages-instructor/instructor-audit-logs-page/instructor-audit-logs-page.component.html

```diff
@@ -0,0 +1,93 @@
+<h1>Student Activity Logs</h1>
+<p>This page allows you to find when your students have accessed or submitted a particular feedback session for a given course. The earliest date you can search for is <b>30 days</b> before today.</p>
```

Comment:
Maybe we'd want to use `LOG_RETENTION_DAYS` here instead of 30 for consistency? I'm also wondering if there's a difference between this constant and `LOGS_RETENTION_PERIOD` (now also 30) as defined in `Const.java` -- would we want these periods to be the same, or do they have a reason to be different?

Reply:
No, there is no difference between the constant. They refer to the same thing (logs bucket retention). For now, have synced up the two by making backend single source of truth